### PR TITLE
Rd 1807 add aws cli

### PR DESCRIPTION
**Pull request by AdarShaked**

### CODE REVIEW

File: cloudify_helm/constants.py [hunk 1]
```diff
@@ -20,14 +20,18 @@
...
+AWS_CLI_VENV = "aws_cli_venv"
 CONFIGURATION = "configuration"
 CLIENT_CONFIG = "client_config"
-AUTHENTICATION = 'authentication'
+AUTHENTICATION = "authentication"
 RESOURCE_CONFIG = "resource_config"
 EXECUTABLE_PATH = "executable_path"
 DATA_DIR_ENV_VAR = "HELM_DATA_HOME"
 CACHE_DIR_ENV_VAR = "HELM_CACHE_HOME"
+AWS_CLI_TO_INSTALL = "awscli==1.19.35"
```
Comment (EarthmanT) [inline cloudify_helm/constants.py:31]: What's significant about this version?

File: cloudify_helm/constants.py [hunk 2]
```diff
@@ -20,14 +20,18 @@
...
 CONFIG_DIR_ENV_VAR = "HELM_CONFIG_HOME"
 USE_EXTERNAL_RESOURCE = "use_external_resource"
 HELM_ENV_VARS_LIST = [DATA_DIR_ENV_VAR, CACHE_DIR_ENV_VAR,
                       CONFIG_DIR_ENV_VAR]
+AWS_ENV_VAR_LIST = ['AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY',
+                    'AWS_DEFAULT_REGION']
```
Comment (EarthmanT) [inline cloudify_helm/constants.py:37]: Why double quotes everywhere else and single quotes here?

File: cloudify_helm/constants.py (hunk 1 above)
Comment (AdarShaked) [reply cloudify_helm/constants.py:31]: They are going to deprecate for python2 so just to be on the safe side.
https://docs.aws.amazon.com/cli/latest/userguide/install-linux.html

File: cloudify_helm/constants.py (hunk 2 above)
Comment (AdarShaked) [reply cloudify_helm/constants.py:37]: Mistake

**Review by EarthmanT — APPROVED** [review]

**Pull request merged**